recipes/minestrone/LICENSE

```diff
@@ -0,0 +1,9 @@
+MIT License
```

Comment:
Where'd this come from? I don't see it in the repo anywhere.

Reply:
I think they neglected to include the file even though on PyPI it's noted it uses MIT. I've PRed to include the license.

Reply:
Ok. We'll have to wait for that to be accepted.

Reply:
It has been accepted @dopplershift https://github.com/adamghill/minestrone/pull/2#event-7594475864

Reply:
👍